client/packages/cli/README.md

```diff
@@ -149,6 +215,16 @@ Below is a detailed explanation of the estimation results for a blockchain trans
 
 Please note that these estimations are subject to change based on the state of the blockchain network at the time of the transaction, and they serve as a guide to understanding the potential costs and rewards associated with a transaction.
 
+Examples:
+```
+# Estimate the max reward of an asset transfer DOT -> ETH on Seploia target with a given over spend percent
+p cli estimate-max-reward --action tass --base-asset dot --target sepl --target-asset eth --target-amount 10 --over-spend 0.1 -o 'fast'
```

Comment:
From the output of this cli I understand that maxReward is executorFeeEstimate + estimatedValue
What I dont understand is why executorFeeEstimate is 3.7 DOT which is around 12$ atm.